storage/storage_model.go

```diff
@@ -68,12 +68,24 @@ func (info *LogChunkInfo) fromKey(path string) error {
 	return nil
 }
 
+func testIdFromKey(path string) (string, error) {
+	keyParts := strings.Split(path, "/")
+	if strings.Contains(path, "/tests/") && len(keyParts) >= 5 {
+		return keyParts[4], nil
+	}
+	return "", errors.Errorf("Path %s does not contain a well-formed test id prefix", path)
```

Comment:
I think this should be a programmatic error since this code creates appends path prefixes.
`programmatic error: unexpected test ID prefix in path  '%s'`

Reply:
Is there a guide on what that means?